test/unit/ngsiv2/plugins/compress-timestamp-plugin_test.js

```diff
@@ -118,7 +118,7 @@ const iotAgentConfig = {
     providerUrl: 'http://smartgondor.com'
 };
 
-describe('NGSI-v2 - Timestamp compression plugin', function () {
+xdescribe('NGSI-v2 - Timestamp compression plugin', function () {
```

Comment:
xdescribe() should be solved before this PR gets merged.

Reply:
3 cases in this file.

Reply:
removed. Deprecated functionaliy